samila/params.py

```diff
@@ -48,7 +48,7 @@
 COLOR_SIZE_ERROR = "Color list size is not equal to the data size."
 SAVE_NO_DATA_ERROR = "Data file can't be saved. At least one of the data1 or data2 is None."
 INVALID_COLOR_TYPE_ERROR = "Given color/bgcolor type is not supported."
-VERSION_WARNING = "Your plots may differ as the version of matplotlib ({0}), Python ({1}), or Samila ({2}) that you are using is not the same as the source."
+VERSION_WARNING = "Source matplotlib version({0}) or Python version({1}) is different from yours, plots may be different."
```

Comment:
I think `release` is not originated from the latest `dev`.

Reply:
It's something related to `PyCharm` cache!!
Fixed in https://github.com/sepandhaghighi/samila/pull/187/commits/5f074bfbde4ca56dccd3d1019a9aff926b72b21d